README.md

```diff
@@ -68,7 +68,7 @@ $ sudo apt-get install python3-venv python3-wheel
 $ python3 -m venv .venv
 $ . .venv/bin/activate
 $ pip install -U pip
-$ git clone https://github.com/rust-ops/homu.git
+$ git clone https://github.com/rust-lang/homu.git
```

Comment:
I also notice this issue. @Mark-Simulacrum could you take a look at this PR? (notifying because of the last committer on master)